oxd-server/src/main/resources/swagger.yaml

```diff
@@ -206,10 +208,22 @@ paths:
                     example: https://as.gluu.org/
                   exp:
                     type: integer
-                    example: 299
+                    format: int64
+                    description: number of milliseconds since January 1 1970 UTC, indicating when this token will expire
+                    example: 1335205592410
                   iat:
                     type: integer
-                    example: 1419350238
+                    format: int64
```

Comment:
I guess integer is enough to show seconds, no?

Reply:
Well, if it is seconds, integer should be enough for next 19 years. 
Integer i = Integer.MAX_VALUE - 1535709072; // 1535709072 current count of  seconds since 1970
i ==> 611774575
That is roughly 19.386 years.